common/appointment/create.ts

```diff
@@ -149,3 +172,16 @@ export const createZoomMeetingForAppointments = async (
         throw new Error(`Zoom - Error while creating zoom meeting: ${e}`);
     }
 };
+
+export const saveZoomMeetingReport = async (appointment: Lecture) => {
+    const meetingReports: Prisma.JsonValue[] = appointment.zoomMeetingReport || [];
```

Comment:
Variables not needed?

Reply:
which variables do you mean?

Reply:
which variables do you mean? @Jonasdoubleyou